app/api/hello/atg.ts

```diff
@@ -0,0 +1,337 @@
+import { gql, useSubscription } from "@apollo/client";
```

Comment:
only keep what are required in the component Things like curl query etc can be removed so that we can review easily. This file has to be in services.